neat-form-core/src/main/java/com/nerdstone/neatformcore/views/builders/MultiChoiceCheckBoxViewBuilder.kt

```diff
@@ -85,6 +103,10 @@ class MultiChoiceCheckBoxViewBuilder(override val nFormView: NFormView) : ViewBu
             if (nFormSubViewProperty.isExclusive != null && nFormSubViewProperty.isExclusive == true) {
                 setTag(R.id.is_exclusive_checkbox, true)
             }
+
+            if (checkBoxTextSize != null) {
```

Comment:
Using `  checkBoxTextSize?.also { 
                textSize = it
            }
` is safer; will just ignore it if checkboxTextSize is null